_data/releases.yml

```diff
@@ -4378,3 +4378,26 @@
     docker_arm: true
   source: true
   previous_release: v23.1.0
+  withdrawn: true
+
+- release_name: v23.1.2
+  major_version: v23.1
+  release_date: '2023-05-25'
+  release_type: Production
+  go_version: go1.19
+  sha: 810d4f27a7f02b9cc2750cab654ed1c62ac3e75a
+  has_sql_only: true
+  has_sha256sum: true
+  mac:
+    mac_arm: true
+  windows: true
+  linux:
+    linux_arm: true
+    linux_intel_fips: true
+    linux_arm_fips: false
+  docker:
+    docker_image: cockroachdb/cockroach
+    docker_arm: true
+  source: true
+  previous_release: v23.1.1
```

Comment:
Does this need to reference a non-withdrawn version?

Reply:
I don't think so?
v23.1.1 still references v23.1.0 even though .0 was withdrawn